README.md

```diff
@@ -46,6 +47,8 @@ version 2.0][license-link].
 [codecov-link]: https://codecov.io/gh/mozilla-lockbox/lockbox-extension
 [waffle-image]: https://badge.waffle.io/mozilla-lockbox/lockbox-extension.svg?columns=In%20Progress
 [waffle-link]: https://waffle.io/mozilla-lockbox/lockbox-extension
+[greenkeeper-image]: https://badges.greenkeeper.io/mozilla-lockbox/lockbox-extension.svg
+[greenkeeper-link]: https://greenkeeper.io/
```

Comment:
Is there a more-specific link for us to use here? I'm not sure if Greenkeeper has a summary page for us...

Reply:
Greenkeeper doesn't have summary pages for project.